src/test/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/AlignmentIntervalUnitTest.java

```diff
@@ -59,7 +59,7 @@ public void testAlignmentIntervalOverlap() throws Exception {
      *                                      intended to be used for testing concordance between the two constructors)
      */
     @DataProvider(name = "AlignmentIntervalCtorTestForSimpleInversion")
-    Object[][] createInputsAndExpectedResults_BwaMemAlignmentConstruction() {
+    public static Object[][] createInputsAndExpectedResults_BwaMemAlignmentConstruction() {
```

Comment:
The problem you're trying to solve is actually caused by bogus data generation by this unit test.
BWA would never create a primary alignment with hard clips, but this class does.  The problem would go away if you changed line 76 to these 4 lines:
```
            int samFlag = 0;
            if ( !strandedness[i] ) samFlag = SAMFlag.READ_REVERSE_STRAND.intValue();
            if ( cigarStrings[i].indexOf('H') != -1 ) samFlag |= SAMFlag.SUPPLEMENTARY_ALIGNMENT.intValue();
            final BwaMemAlignment bwaMemAlignment = new BwaMemAlignment(samFlag,
```
You could argue that BwaMemAlignmentUtils ought to do the right thing, anyway.  So, if you want to make the change to BwaMemAlignmentUtils, go right ahead.